processing/src/main/java/io/druid/segment/filter/DimensionPredicateFilter.java

```diff
@@ -23,6 +23,7 @@
 import com.google.common.base.Predicate;
 import com.metamx.collections.bitmap.ImmutableBitmap;
 import com.metamx.common.guava.FunctionalIterable;
+import io.druid.query.extraction.ExtractionFn;
```

Comment:
@drcrallen changed this per @gianm's suggestion to have the extractionFn applied in the DimensionPredicateFilter